src/endpoints/network/entities/networkConfig.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class NetworkConfig {
```

Comment:
Please change filename to `'network.config.ts'` to keep same structure.
Thank you!